.travis.yml

```diff
@@ -16,7 +16,7 @@ env:
   global:
     - APM_SERVER_PORT=8200
     - APM_SERVER_URL=http://localhost:8001
-    - MODE=saucelabs
+    - MODE=none
```

Comment:
Can you add `- DISPLAY=:0` to fix the firefox errors in travis. Currently the tests does not run on travis.